### PR TITLE
Rosetta pguser addition

### PR DESCRIPTION
**Pull request by yazu-codes**

### CODE REVIEW

File: hedera-mirror-importer/src/main/resources/db/scripts/init.sql [hunk 1]
```diff
@@ -15,8 +17,16 @@ create user :grpc_user with login password :'grpc_password';
 
 grant connect on database :db_name to :grpc_user;
 
+create user :rosetta_user with login password :'mirror_rosetta_pass';
```
Comment (Daniel-K-Ivanov) [inline hedera-mirror-importer/src/main/resources/db/scripts/init.sql]: I think that this should be `rosetta_password` not `mirror_rosetta_pass`

**Review by Daniel-K-Ivanov — CHANGES_REQUESTED** [review]
Suggestion: add "if not exist" in the statements for the user creation

**Comment by failfmi** [comment]
Target branch is a bit old, please fork [hashgraph:rosetta](https://github.com/hashgraph/hedera-mirror-node/tree/rosetta) and switch to it.

File: docs/installation.md [hunk 2]
```diff
@@ -23,14 +23,14 @@ compile a runnable Mirror Node JAR file in the `target` directory.
 
 In addition to OpenJDK 11, you will need to install [PostgreSQL](https://postgresql.org) 9.6 and initialize it. The only
 setup required is to create the initial database and owner since [Flyway](https://flywaydb.org) manages the
-database schema. The SQL script located at `hedera-mirror-importer/src/main/resources/db/scripts/init.sql` can be used to
+database schema. The SQL script located at `hedera-mirror-importer/src/main/resources/db/scripts/0_init.sql` can be used to
```
Comment (failfmi) [inline docs/installation.md:26]: The SQL scripts located at `hedera-mirror-importer/src/main/resources/db/scripts/init` can be used to...

File: docs/installation.md [hunk 3]
```diff
@@ -23,14 +23,14 @@ compile a runnable Mirror Node JAR file in the `target` directory.
...
 accomplish this. Edit the file and change the `db_name`, `db_user`, `db_password` `db_owner`, `grpc_user`, or
 `grpc_password` as appropriate.
```
Comment (failfmi) [inline docs/installation.md]: `rosetta_user`, or `rosetta_password` as appropriate.

File: docs/installation.md [hunk 4]
```diff
@@ -23,14 +23,14 @@ compile a runnable Mirror Node JAR file in the `target` directory.
...
 Make sure the application [configuration](configuration.md) matches the values in the script. Run the script as a DB
 admin user and check the output carefully to ensure no errors occurred.
 
 ```console
-psql postgres -f hedera-mirror-importer/src/main/resources/db/scripts/init.sql
+psql postgres -f hedera-mirror-importer/src/main/resources/db/scripts/0_init.sql
```
Comment (failfmi) [inline docs/installation.md:33]:
```
psql postgres -f hedera-mirror-importer/src/main/resources/db/scripts/init/0_init.sql
psql postgres -f hedera-mirror-importer/src/main/resources/db/scripts/init/1_grants.sql
```
or a way to execute the `init` directory as a whole

File: hedera-mirror-importer/src/main/resources/db/scripts/init/0_init.sql [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+\set grpc_user 'mirror_grpc'
+\set grpc_password 'mirror_grpc_pass'
+\set rosetta_user 'mirror_rosetta'
+\set rosetta_password 'mirror_rosetta_pass'
+
+select 'create database mirror_node' where not exists (select from pg_database where datname = :'db_name')\gexec
+
+do $$
+begin
+  create user mirror_node with SUPERUSER password 'mirror_node_pass';
```
Comment (failfmi) [inline hedera-mirror-importer/src/main/resources/db/scripts/init/0_init.sql:16]: Not everything here uses the variables above

**Review by failfmi — APPROVED** [review]

**Review by failfmi — APPROVED** [review]

**Review by Daniel-K-Ivanov — APPROVED** [review]

**Pull request merged**